uw-frame-components/portal/main/controllers.js

```diff
@@ -100,12 +102,18 @@ define(['angular', 'require'], function(angular, require) {
         $rootScope.GuestMode = true;
       } else {
         // Get first letter of first name or display name
-        var username =
-          vm.user.firstName ? vm.user.firstName : vm.user.displayName;
-        if (username === '' || !angular.isString(username)) {
-          vm.firstLetter = '?';
+        if ($sessionStorage.optAvatar) {
```

Comment:
So this would store this opt-out-of-avatar preference client-side in `$sessionStorage?` Maybe better to use `KeyValueStore` to store the preference server-side so it'll be honored across all the user's usages?